### ISSUE TITLE
Vectorize layer

### ISSUE THREAD

**Issue by circular17** [bgrabitmap/lazpaint#76]
Make polygons out of reduced color palette

**Comment by lainz** [comment]
I can remember when I know nothing about graphic design, that I downloaded a tool made just only for that purpouse, to vectorize a bitmap, and was like the most amazing thing ever.

**Comment by circular17** [comment]
Yeah that’s cool. I have an idea how to do it. First to reduce the number of color, like when saving in 16 colours. Then to make a layer mask for each color, then vectorize each layer, and finally make one polygon with each color.
